bot/cogs/fun.py

```diff
@@ -199,6 +199,43 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @commands.command(aliases=["leet"])
+    async def Leetify(self, ctx):
```

Comment:
Purely commiting this won't help you as you're overwriting the `context` below this, you'll need to actually adjust the code you're using to use this `content` instead of getting it from your list comprehention:
```py
[i for i in ctx.message.content.split(" ") if i.lower() != ">leet" and i.lower() != ">leetify"]
```